docs/docs/getting-started/compatibility/supported-hardware.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 3
+title: Supported hardware
+---
+
+# Supported hardware
+
+The AICA System supports many different types of manipulators and sensors. The following tables present these
+supported devices, along with necessary prerequisites and additional software that these might require.
+
+## Robotic manipulators
+
+| Brand        | Supported Devices            | Requirements | Additional Notes | 
+|--------------|-----------------------------|------------------------------|------------------|
+| [ABB](../../examples/guides/abb-guide.md) | IRB 1010, CRB 15000 - 12 | RobotStudio, EGM | - |
+| Doosan | A0509, M0609, M1509 | **TBA** | **TBA** |
+| FANUC | CRX-10𝑖A | **TBA** | **TBA** |
+| Franka | Emika Panda | **TBA** | **TBA** |
+| JAKA | Zu12 | **TBA** | **TBA** |
+| Kassow | KR810, KR1410 | **TBA** | **TBA** |
+| Kinova | Gen3 | **TBA** | **TBA** |
+| KUKA | KR 10 R1100-2, KR 16, KR 70, KR 210 R2700-2 | **TBA** | **TBA** |
+| Mecademic | Meca500 | - | TCP/IP & EtherCAT interfaces available |
+| Stäubli | TX2-40, TX2-60 | **TBA** | **TBA** |
+| UFACTORY | X-ARM 6 | **TBA** | **TBA** |
+| Universal Robots | UR5e, UR10e, UR16e, UR20, UR30 | Polyscope version > 5.X, External control URCap | - |
```

Comment:
External control URCap is also optional, right? (in the sense that perhaps we add as a note?)